pnpm-lock.yaml

```diff
@@ -5980,6 +6006,14 @@ packages:
       '@wallet-standard/features': 1.0.3
     dev: false
 
+  /@solana/wallet-standard-features@1.2.0:
+    resolution: {integrity: sha512-tUd9srDLkRpe1BYg7we+c4UhRQkq+XQWswsr/L1xfGmoRDF47BPSXf4zE7ZU2GRBGvxtGt7lwJVAufQyQYhxTQ==}
+    engines: {node: '>=16'}
+    dependencies:
+      '@wallet-standard/base': 1.0.1
+      '@wallet-standard/features': 1.0.3
+    dev: false
```

Comment:
We're resolving multiple versions of some packages now, it might be something on our end. Looking into it.

Reply:
Awesome, lmk if I can help